recipe/meta.yaml

```diff
@@ -1,25 +1,28 @@
 {% set name = "sqlalchemy" %}
-{% set version = "2.0.21" %}
+{% set version = "2.0.22" %}
 
 package:
   name: sqlalchemy
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/SQLAlchemy-{{ version }}.tar.gz
-  sha256: 05b971ab1ac2994a14c56b35eaaa91f86ba080e9ad481b20d99d77f381bb6258
+  sha256: 5434cc601aa17570d79e5377f5fd45ff92f9379e2abed0be5e8c2fba8d353d2b
 
 build:
-  number: 1
+  number: 0
   script: {{ PYTHON }} -m pip install . --no-deps -vv
 
 requirements:
   build:
     - python                                 # [build_platform != target_platform]
     - cross-python_{{ target_platform }}     # [build_platform != target_platform]
+    - cython >=0.29.24                       # [build_platform != target_platform]
     - {{ compiler('c') }}
   host:
     - python
+    - setuptools >=47
+    - cython >=0.29.24
```

Comment:
I'm not sure why we haven't needed cython up to now.  It's listed for non-PyPy builds:
https://github.com/sqlalchemy/sqlalchemy/blob/rel_2_0_22/pyproject.toml#L5
We can skip it for PyPy if need be but I didn't want to add the complication if it's harmless to have for PyPy.